pkg/build/gobuild.go

```diff
@@ -123,6 +131,7 @@ func (gbo *gobuildOpener) Open() (Interface, error) {
 			buildToDiff: map[string]buildIDToDiffID{},
 			diffToDesc:  map[string]diffIDToDescriptor{},
 		},
+		semaphore: semaphore.NewWeighted(int64(gbo.jobs)),
```

Comment:
Hmm, do we `Open()` once per `ko` invocation or once per build? 🤔

Reply:
It's once per "builder", which means the `--jobs` isn't shared across build _configs_... which is annoying, but I'm happy to fix that if someone actually complains...